sdk/metric/config.go

```diff
@@ -37,7 +37,9 @@ func (c config) readerSignals() (forceFlush, shutdown func(context.Context) erro
 	var fFuncs, sFuncs []func(context.Context) error
 	for _, r := range c.readers {
 		sFuncs = append(sFuncs, r.Shutdown)
-		fFuncs = append(fFuncs, r.ForceFlush)
+		if f, ok := r.(interface{ ForceFlush(context.Context) error }); ok {
+			fFuncs = append(fFuncs, f.ForceFlush)
+		}
```

Comment:
Should we document this functionality e.g. by adding following comment

```
// ForceFlush calls ForceFlush(context.Context) error
// on all Readers that implements this method.
```

to `func (mp *MeterProvider) ForceFlush(ctx context.Context) error {` ?

Reply:
LGTM